decidim-proposals/app/controllers/decidim/proposals/admin/participatory_texts_controller.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Proposals
+    module Admin
+      # This controller manages the participatory texts area.
+      class ParticipatoryTextsController < Admin::ApplicationController
+        helper_method :proposal
+        helper ParticipatoryTextsHelper
+
+        def index
+          @drafts = Proposal.where(component: current_component).drafts.order(:position)
+          @preview_form = form(Admin::PreviewParticipatoryTextForm).instance
+          @preview_form.from_models(@drafts)
+        end
+
+        def new_import
+          enforce_permission_to :import, :participatory_texts
+          @import = form(Admin::ImportParticipatoryTextForm).instance
+        end
+
+        def import
```

Comment:
Shouldn't this be `create` and `new_import` `new`?

Reply:
No, it does not `create`, it imports (by adding) all the proposals in the text to existing proposals, if any.

Reply:
Oh, ok.